app/main/posts/common/post-actions.service.js

```diff
@@ -31,6 +31,26 @@ function (
         },
         getStatuses: function () {
             return ['published', 'draft', 'archived'];
+        },
+        filterPostEditorCategories: function (attributeOptions, categories) {
```

Comment:
moved here because the post-editor vs post-data-editor thing where we duplicate logic was driving me crazy

Reply:
:tada: